esi/src/lib.rs

```diff
@@ -504,7 +517,97 @@ fn event_receiver(
                 except_task,
             });
         }
-        Event::XML(event) => {
+        Event::ESI(Tag::Assign { name, value }) => {
+            // TODO: the 'name' here might have a subfield, we need to parse it
```

Comment:
Is this something that needs to be addressed before release?

Reply:
nah, I think that can wait for the parser refactor